lib/tasks/reports.rake

```diff
@@ -33,20 +38,20 @@ Available conditions:
   * email            => override default email addresses
 
   Example:
-    # Sends out a summary email for the last 3 days.
-    rake reports:summarize days=3 RAILS_ENV="production" # Sends out a summary email for the last 3 days.
+    # Sends out a daily summary email for the last 3 days.
+    rake reports:daily days=3 RAILS_ENV="production" # Sends out a summary email for the last 3 days.
```

Comment:
I'm not sure we should keep these examples here, particularly as the first ones, since the default behaviour of reports:daily/weekly/monthly is what people should be running without changing `days` (I have no idea what that would do).